lms/djangoapps/teams/static/teams/js/models/topic.js

```diff
@@ -20,6 +21,13 @@
             isInstructorManaged: function() {
                 var topicType = this.get('type');
                 return topicType === 'public_managed' || topicType === 'private_managed';
+            },
+
+            getMaxTeamSize: function(courseMaxTeamSize) {
+                if (this.isInstructorManaged()) {
```

Comment:
If a team is instructor managed, are we allowing for unlimited team size?

Reply:
as far as the UI is concerned, yes